### ISSUE TITLE
Expose Keras Regularization Parameters

### ISSUE THREAD

**Issue by Natalie-Caruana** [equinor/gordo#378]
Currently we don't expose any Keras regularization arguments.  The default regularization arguments are being used (which are None), except for the feedforward autoencoder which is hardcoded to the L1 regularizer.

It would be useful to be able to set such arguments in the config file.

**Comment by epa095** [comment]
Same as #127 ?

**Comment by ryanjdillon** [comment]
As per Natalie, first check with Keras FeedForward/LSTM API to see what regularizers can be passed as arguments and which should be passed as layers.

The known example of a regularlizer that should be passed as a layer are all types of [dropout](https://keras.io/layers/core/#Dropout). A possible solution would be a flag parameter like:
```
encoding_dropout:Tuple[bool, ...] = (True, True, False)
```
This could possibly be its own issue. I'll start with the other portion first.
